modules/auxiliary/gather/censys_search.rb

```diff
@@ -46,12 +46,12 @@ def search(keyword, search_type)
         'query' => keyword
       }
 
-      @cli = Rex::Proto::Http::Client.new('www.censys.io', 443, {}, true)
+      @cli = Rex::Proto::Http::Client.new('search.censys.io', 443, {}, true)
       @cli.connect
 
       response = @cli.request_cgi(
         'method' => 'post',
-        'uri' => "/api/v1/search/#{search_type}",
+        'uri' => "/api/v2/hosts/search/#{search_type}",
```

Comment:
Don't think this is correct. If search_type is meant to be the type of search then we have `/v1/search/certificates`.

Given it sounds like v1 will only be around until they replace their v1 APIs I'm almost tempted not to make too many changes until the new API is finalized given that they seem to be adopting this weird strategy of rolling out partial updates to their API as it is completed vs publishing a finalized static API once its ready :/

Reply:
Actually yeah so upon further inspection this is set at line 166, and is taken from the user controllable CENSYS_SEARCHTYPE option on line 31, which can be of values `certificates`, `ipv4` or `websites`.

With the exception of `certificates` I can't find any current APIs, both on the v1 or v2 API, which are replacements for `ipv4` or `websites`.